src/org/opendatakit/briefcase/util/FileSystemUtils.java

```diff
@@ -74,11 +78,46 @@
   static final String ENCRYPTED_FILE_EXTENSION = ".enc";
   static final String MISSING_FILE_EXTENSION = ".missing";
 
+  static final File cacheFile = new File(getFormsFolder().getParent(), "cache.ser");
```

Comment:
We should probably name this something clearer? Maybe `Form Definitions.cache` or `forms.cache`. Whatcha think @lognaturel?